workflow/README.md

```diff
@@ -28,19 +26,70 @@ SPACEPSWD="yourpassword"
 Don't commit these changes to the repo.
 <!-- TODO: Insecure. Make this import from an environment file or the keychain. -->
 
+#### Simple case: single target
+
+Make a new configuration file with the region and time period you want to analyse. All the possible parameters are listed in [rose-suite.conf](./rose-suite.conf). You can see examples in [example](./example/). Command line usage examples are shown in [example-cylc-calls.sh](./example-cylc-calls.sh).
+
```

Comment:
Is there a place where the individual variables in the conf files are explained? For example, it's not obvious what order the values in the bounding box are provided in.

Reply:
That's a good point. I've consolidated all of the descriptions of the variables in`meta/rose-meta.conf` and updated the rose-suite.conf file to simplify it accordingly. There are a couple of key descriptive comments in there still, but most have moved out. Does that work better for you?